nototools/hb_input_test.py

```diff
@@ -135,76 +135,84 @@ class definition end
 '''
 
 
+def make_font(feature_source, fea_type='fea'):
+    """Return font with GSUB compiled from given source.
+
+    Adds a bunch of filler tables so the font can be saved if needed, for
+    debugging purposes.
+    """
+
+    font = makeTTFont()
+    glyph_order = font.getGlyphOrder()
+
+    font['cmap'] = cmap = newTable('cmap')
+    table = cmap_format_4(4)
+    table.platformID = 3
+    table.platEncID = 1
+    table.language = 0
+    table.cmap = {AGL2UV[n]: n for n in glyph_order if n in AGL2UV}
+    cmap.tableVersion = 0
+    cmap.tables = [table]
+
+    font['glyf'] = glyf = newTable('glyf')
+    glyf.glyphs = {}
+    glyf.glyphOrder = glyph_order
+    for name in glyph_order:
```

Comment:
Yea, the function imported from feaLib creates a test font with a hundred or so dummy glyphs.